[2025-09-08T17:10:23.904+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:10:23.911+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskD backfill__2025-04-12T12:00:00+00:00 [queued]>
[2025-09-08T17:10:23.916+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskD backfill__2025-04-12T12:00:00+00:00 [queued]>
[2025-09-08T17:10:23.916+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:10:23.929+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): taskD> on 2025-04-12 12:00:00+00:00
[2025-09-08T17:10:23.933+0000] {standard_task_runner.py:72} INFO - Started process 21335 to run task
[2025-09-08T17:10:23.936+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskD', 'backfill__2025-04-12T12:00:00+00:00', '--job-id', '179', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpsmim1b12']
[2025-09-08T17:10:23.937+0000] {standard_task_runner.py:105} INFO - Job 179: Subtask taskD
[2025-09-08T17:10:23.964+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskD backfill__2025-04-12T12:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:10:24.011+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskD' AIRFLOW_CTX_EXECUTION_DATE='2025-04-12T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-04-12T12:00:00+00:00'
[2025-09-08T17:10:24.011+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:10:24.022+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-08T17:10:24.023+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK D has executed!']
[2025-09-08T17:10:24.030+0000] {subprocess.py:86} INFO - Output:
[2025-09-08T17:10:24.031+0000] {subprocess.py:93} INFO - TASK D has executed!
[2025-09-08T17:10:24.032+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-08T17:10:24.045+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:10:24.045+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskD, run_id=backfill__2025-04-12T12:00:00+00:00, execution_date=20250412T120000, start_date=20250908T171023, end_date=20250908T171024
[2025-09-08T17:10:24.068+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:10:24.078+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:10:24.079+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
